business/services.go

```diff
@@ -243,15 +243,19 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 	}()
 
 	var vsCreate, vsUpdate, vsDelete bool
-	go func() {
-		defer wg.Done()
-		vsCreate, vsUpdate, vsDelete = getPermissions(in.k8s, namespace, kubernetes.VirtualServices)
-	}()
-
 	var drCreate, drUpdate, drDelete bool
 	go func() {
 		defer wg.Done()
-		drCreate, drUpdate, drDelete = getPermissions(in.k8s, namespace, kubernetes.DestinationRules)
+		/*
+			We can safely assume that permissions for VirtualServices will be similar as DestinationRules.
+
+			Synced with:
+			https://github.com/kiali/kiali-operator/blob/master/roles/default/kiali-deploy/templates/kubernetes/role.yaml#L62
```

Comment:
just FYI - this link will eventually grow stale because its linking to "master" rather than a fixed tag. Probably should have used "v1.36.0".